### PR TITLE
chore: Add missing public key

### LINKED ISSUE #26: sigstore cannot find keyring in release pipeline
### Helm chart

foreman

### Helm chart version

All

### Actual behavior (issue description)

The sigstore plugin fails loading our public keyring when publishing.

### Expected behavior

_No response_

### Steps to reproduce

None

### Custom configuration

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by lusu007**
## Description

Fixes #26 

Fore more information consult https://github.com/sigstore/helm-sigstore/blob/main/USAGE.md#general-options

## Additional Context

N/A

## Checklist

- [ ] The chart version in Chart.yaml has been updated according to semantic versioning (semver). This update is not required if the changes only impact README.md files.
- [ ] All variables are document in the Chart's values.yaml and README.md files.
- [x] The pull request title meets the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification and includes the chart name, for example: `feat(chart-name): Add replica support`

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -31,10 +31,12 @@ jobs:
           IFS=""
           echo "$PGP_PRIVATE_KEY" | gpg --dearmor > $HOME/secring.gpg
           echo "$PGP_PASSPHRASE" > $HOME/passphrase.txt
+          echo "$PGP_PUBLIC_KEY" > $HOME/pubring.gpg
```
Comment (meyfa) [inline .github/workflows/release.yml:34]: Where is `PGP_PUBLIC_KEY` set?

Comment (lusu007) [reply .github/workflows/release.yml:34]: It is added via the actions secrets

Comment (lusu007) [reply .github/workflows/release.yml:34]: We could also download it from keybase.io

Comment (meyfa) [reply .github/workflows/release.yml:34]: In that case, I think it must be added to `env` to be available.

**Review by meyfa — APPROVED** [review]

**Pull request merged**